assets/sass/base/links.scss

```diff
@@ -0,0 +1,11 @@
+///////////
+// Links //
+///////////
+
+a, a:visited {
+    text-decoration: none;
+}
+
+a:not([class]) {
```

Comment:
It's unclear what this does to someone who didn't write this. Could use comments/docblock